Rootlog_V2.md

```diff
@@ -748,9 +748,9 @@ Common sequences of multiple actions in a row.
 - `Sp->9/(w+Ew+Lw)9->` *Furious Protector comes into town.*
 - `#bankO->/w->$` *The Riverfolk use Export to gain a warrior in Payments. Note: Don't use the "craft" notation in this case, since the card isn't added to Crafted Improvements and the item isn't added to the faction board.*
 - `%r->e/CXO12/(2Cw+3Ow)12->/$_O+$_C->h/++3` *The Vagrant Vagabond instigates a battle between the Marquise and the Riverfolk.*
-- `t_t7_10_11->10/t_t10^2/#faith$_2->` - *The Keepers delve a relic (possibly foolishly).*
+- `t_t7_10_11->10/t10^t_2_t/#faith$_2->` - *The Keepers delve a relic (possibly foolishly).*
 - `h_Op->4/h_OXD4/3Dw4->` *The Flotilla battles the Duchy.*
-- `t_f5->$/++5` - *The Keepers recover a 3 point relic which also completes a relic column.*
+- `t_3_f5->$/++5` - *The Keepers recover a 3 point relic which also completes a relic column.*
```

Comment:
Same question here about `t_3_f5`; maybe I'm missing something!

Reply:
You didn't miss something. I forgot to edit the part about optional notation for relic value, which I have just rectified. The `t_f_3` notation creates a problem when reading a recover action like `t_f_35->$`. I know that the current version of the Keepers notation is to infer the value of the relic from a subsequent score points action. I think that it makes the notation less readable.